### PR TITLE
Update Release Actions

### PR DESCRIPTION
**Pull request by Michael-F-Bryan**
Ties up some loose ends after #301.

### CODE REVIEW

**Comment by Michael-F-Bryan** [comment]
@saidinesh5 it looks like we can't build our Docker image because it needs access to the Docker daemon and there is no such thing as `docker build -v /run/docker.sock`. 

Do you have any suggestions for how we can package `rune` in a Dockercontainer *and* keep `librunecoral`?

**Comment by saidinesh5** [comment]
I have fixed this by manually building librunecoral in Dockerfile and setting the `RUNECORAL_DIST_DIR` environment variable

```
$ docker run -it -v $PWD:$PWD rune rune model-info $PWD/examples/sine/sinemodel.tflite
Ops: 3
Inputs:
        dense_2_input: f32[1,1]
Outputs:
        Identity: f32[1,1]
```

Only catch is the dockerfile is specific to x86_64 now. let me know if we want to have that for Arm and other architectures too

**Comment by Michael-F-Bryan** [comment]
> Only catch is the dockerfile is specific to x86_64 now. let me know if we want to have that for Arm and other architectures too

That should be fine for now. I didn't even know Docker supported ARM containers.

**Comment by saidinesh5** [comment]
I have started building cross containers for our use case today..  and disabling docker builds in librunecoral's makefile by default. So do not merge this yet, fixes will have to come from there..

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
-RUN apt-get update -y && apt-get -y install libclang-dev clang curl build-essential git 
+RUN apt-get update -y && apt-get -y install libclang-dev clang curl build-essential git python3-numpy zip unzip curl wget cmake pkg-config
+
+ARG BAZEL_VERSION=4.0.0
+RUN wget -O /bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
+    bash /bazel && \
+    rm -f /bazel
+
+WORKDIR /lib
+RUN git clone --recurse-submodules https://github.com/hotg-ai/librunecoral && cd librunecoral && \
```
[the remark this answers is not in the document]
Comment (saidinesh5) [reply Dockerfile:11]: Yeah ignore this for now.. setting up cross build containers for librunecoral .. (With bazel and basic build dependencies - no hardware acceleration) . Then we can directly use those containers to build rune and everything will be transparent (hopefully) and these lines will be gone

**Pull request merged**